README.rst

```diff
@@ -34,6 +34,23 @@ feedback.
 .. |Scale where good is in the middle| image:: happy_sad_happy_example.png
 .. |Numberical scale| image:: numerical_example.png
 
+The instructors can view reports in their course instructor dashboard. The reports shows the count for every score, the average sentiment score, and the last 10 feedback comments.
+
+To enable the FeedbackXBlock report in the instructor dashboard, add the following to the `openedx-common-settings` tutor patch:
```

Comment:
Please add more details on this point.
I am new to Tutor and I am not getting where to add these settings.
Adding a file hyperlink will be great if that's the case.